src/program/snabbnfv/traffic/traffic.lua

```diff
@@ -41,6 +41,11 @@ function traffic (pciaddr, confpath, sockpath)
          local mtime2 = C.stat_mtime(confpath)
          if mtime2 ~= mtime then
             print("Loading " .. confpath)
+            if mtime2 == 0 then
```

Comment:
This change may have unintended consequences: The traffic process will be started as a daemon in the OpenStack context and initially its configuration file may not exist. This change would seem to cause the traffic process to terminate instead of polling for the configuration to be created.

Can we skip this change or make it in another way that does not change the high-level behavior of the traffic process?